### ISSUE TITLE
Crashes

### ISSUE THREAD

**Issue by lukas** [wandb/client#2]
* Weights and Biases version: latest
* Python version: 2.7
* Operating System: ubuntu

### Description

I installed wandb and typed wandb init

Traceback (most recent call last):
  File "/home/l2k/.local/bin/wandb", line 7, in <module>
    from wandb.cli import cli
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/__init__.py", line 8, in <module>
    from .sync import Sync
  File "/home/l2k/.local/lib/python2.7/site-packages/wandb/sync.py", line 3, in <module>
    from watchdog.observers import Observer
ImportError: No module named watchdog.observers

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by lukas** [comment]
Ok I did:

pip install watchdog
pip install sets

now I get
Traceback (most recent call last):
  File "/Users/lukasbiewald/.pyenv/versions/3.5.2/bin/wandb", line 7, in <module>
    from wandb.cli import cli
  File "/Users/lukasbiewald/.pyenv/versions/3.5.2/lib/python3.5/site-packages/wandb/cli.py", line 4, in <module>
    from sets import Set
ImportError: cannot import name 'Set'

**Comment by lukas** [comment]
I guess Set was deprecated in python 3

http://stackoverflow.com/questions/14855110/importing-sets-in-python

**Comment by vanpelt** [comment]
Fixed in 0.4.1
